test/geocoders/geonames.py

```diff
@@ -55,20 +55,62 @@ def test_query_urlencoding(self):
 
     def test_reverse(self):
```

Comment:
This test contains too many checks. Could you please split it to multiple distinct test methods?